draft-ietf-quic-transport.md

```diff
@@ -2675,201 +2676,289 @@ created, it MUST close the connection with error code STREAM_STATE_ERROR.
 Stream ID 0 (0x0) is reserved for the cryptographic handshake.  Stream 0 MUST
 NOT be used for application data, and is the first client-initiated stream.
 
-A QUIC endpoint MUST NOT reuse a Stream ID.  Streams MUST be created
-in sequential order.  Open streams can be used in any order.  Streams
-that are used out of order result in lower-numbered streams in the
-same direction being counted as open.
+A QUIC endpoint MUST NOT reuse a Stream ID.  Open streams can be used in any
+order.  Streams that are used out of order result in lower-numbered streams in
+the same direction being counted as open.
```

Comment:
There is both direction and type(uni/bidi).  You use 'type' below, which seems fine here as well?

nit: "being opened." not being counted as opened.